cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -0,0 +1,182 @@
+import logging
+import re
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_network_interface_data(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    paginator = client.get_paginator('describe_network_interfaces')
+    subnets = []
+    for page in paginator.paginate():
+        subnets.extend(page['NetworkInterfaces'])
+    return subnets
+
+
+def load_network_interfaces(neo4j_session, data, region, aws_account_id, aws_update_tag):
+
+    elb_associations = []
+    instance_associations = []
+
+    for n_idx, network_interface in enumerate(data):
+
+        matchObj = re.match(r'^ELB (.*)', network_interface.get('Description', ''))
```

Comment:
Can you explain the regex match?

Reply:
I was very suprised seeing this, but this is how AWS notates if and IP belongs to an ELB. There is a similar pattern for EFS as well... for a future enchancement.

More info: https://aws.amazon.com/premiumsupport/knowledge-center/elb-find-load-balancer-IP/ 

I realize now that this works only for classic loadbalancers, I think I will need update this pattern for gen2 ELBs a bit.